### PR TITLE
Feature/add network policies

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: cf_deployment_functions.sh [hunk 1]
```diff
@@ -83,4 +86,30 @@ unmap_blue_route() {
...
+add_network_polices() {
+  suffix=$1
+  cat ${BIN_DIR}/network-policies.properties | grep ${APP_NAME} | while read route
+  do
+    # src and dest must have <<suffix>> replaced according to whether they are the application currently being deployed
+    src=$(echo ${route} | cut -d= -f1)
+    if [ ${src} = "${APP_NAME}<<suffix>>" ]; then
+      src=$(echo ${src} | sed "s/<<suffix>>/$suffix/g")
+    else
+      src=$(echo ${src} | sed "s/<<suffix>>/-${CF_SPACE}/g")
```
Comment (YetAnotherMatt) [inline cf_deployment_functions.sh]: "-${CF_SPACE}" should instead be "${SPACE_SUFFIX}" to work in production too

File: cf_deployment_functions.sh [hunk 2]
```diff
@@ -83,4 +86,30 @@ unmap_blue_route() {
...
+    fi
+
+    dest=$(echo ${route} | cut -d= -f2)
+    if [ ${dest} = "${APP_NAME}<<suffix>>" ]; then
+      dest=$(echo ${dest} | sed "s/<<suffix>>/$suffix/g")
+    else
+      dest=$(echo ${dest} | sed "s/<<suffix>>/-${CF_SPACE}/g")
```
Comment (YetAnotherMatt) [inline cf_deployment_functions.sh]: "-${CF_SPACE}" should instead be "${SPACE_SUFFIX}" to work in production too

File: network-policies.properties [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# this file is in the format
+# source_app=destination_app
+# and is used to create network policies allowing communication between apps
+# it is important that source_app and destination_app reflect the name of the app in the manifest (excluding any ((suffix)) )
+help-to-buy-healthy-foods<<suffix>>=htbhf-claimant-service<<suffix>>
+htbhf-claimant-service<<suffix>>=help-to-buy-healthy-foods<<suffix>>
```
Comment (YetAnotherMatt) [inline network-policies.properties]: We need to remove this second route

File: cf_deployment_functions.sh (hunk 2 above)
Comment (dwybourn) [reply cf_deployment_functions.sh]: Updated

File: network-policies.properties (hunk 3 above)
Comment (dwybourn) [reply network-policies.properties]: Removed.

**Pull request merged**